src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -1156,6 +1176,13 @@ module Types = struct
               ~doc:"Public key specifying which account to unlock"
               ~typ:(non_null public_key_arg) ]
 
+    let register_hardware_wallet =
+      obj "RegisterHardwareWalletInput" ~coerce:Fn.id
+        ~fields:
+          [ arg "hardwareWalletNonce"
```

Comment:
Probably can just be called "nonce", I can't imagine it being ambiguous in this context, given the name of the mutation and the doc comment